sdk-src/src/main/java/com/hpe/adm/nga/sdk/Octane.java

```diff
@@ -88,6 +89,10 @@ public EntityList entityList(String entityName) {
         return OctaneClassFactory.getSystemParamImplementation().getEntityList(octaneHttpClient, getBaseDomainFormat(), entityName);
     }
 
+    public <T extends TypedEntityList> T entityList(Class<T> entityListClass) {
```

Comment:
That would mean generating Octane which would make a lot more generated
I think that it is a good ER but not for now